.readthedocs.yaml

```diff
@@ -0,0 +1,9 @@
+version: 2
```

Comment:
Squeezing this in because I totally missed the [deprecation announcement](https://blog.readthedocs.com/migrate-configuration-v2/)